src/pages/settings/InitialSettingsPage.js

```diff
@@ -357,15 +357,15 @@ function InitialSettingsPage(props) {
                         </PressableWithoutFeedback>
                     </Tooltip>
                     <PressableWithoutFeedback
-                        style={[styles.mt1, styles.mw100]}
+                        style={[styles.mt1, styles.w100, styles.mw100]}
```

Comment:
Do we need both styles.w100 and styles.mw100?

Reply:
Missed this. I don't think max-width 100% is required anymore, since 100% width is fixed. It doesn't hurt to leave both in, however. https://developer.mozilla.org/en-US/docs/Web/CSS/max-width